olearyspicker.py

```diff
@@ -16,20 +17,47 @@ def __init__(self, title, short_title, slug, address, distance, phone, url):
 
 
 def main():
-    distance = "4000"
-    point = "18.0388702%2C59.3213309"
-    r = requests.get("http://olearys.se/api/v1/restaurants/?language_code=sv-se&site_id=2&point={point}&dist={dist}".format(point=point, dist=distance))
+    parser = argparse.ArgumentParser(
+        description="Find and randomize nearby O'learys restaurants. Stockholm central coordinates is 18.0686,59.3293. Gothenburg central is 11.9746,57.7089, I hate argparse formatting"
+    )
+    parser.add_argument('-d', '--distance',
+                        type=int, default='4000',
+                        help='Integer for distance in meters'
+                        )
+    parser.add_argument('-c', '--coordinates',
+                        default='18.0686,59.3293',
+                        help='Coordinates in format {format} (W/E, N/S)'.format(format='18.0388702,59.3213309')
+                        )
+    args = parser.parse_args()
+
+    if args.distance is not None:
+        distance = args.distance
+    else:
+        distance = '4000'
+
+    if args.coordinates is not None:
```

Comment:
(Was tempted to type something like "since you have default values in your argparse parser argument-adding statement arguments, but I quietly bowed out of it...)